lib/config/config_test.go

```diff
@@ -61,6 +95,77 @@ var _ = t.Describe("Config", func() {
 			// Then
 			Expect(err).NotTo(BeNil())
 		})
+
+		It("should fail on unrecognized image volume type", func() {
+			// Given
+			sut.ImageVolumes = invalidPath
+
+			// When
+			err := sut.Validate(nil, false)
+
+			// Then
+			Expect(err).NotTo(BeNil())
+		})
+
+		It("should fail on wrong default ulimits", func() {
+			// Given
+			sut.DefaultUlimits = []string{invalidPath}
```

Comment:
Maybe using a path here is a bit misleading, suggest `[]string{"invalid=-1:-1"}`

Reply:
Applied as suggested.